<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDocumentThumbnailDemoClass</class>
 <widget class="QMainWindow" name="CDocumentThumbnailDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDocumentThumbnailDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnText">
        <property name="text">
         <string>Text Thumbnail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImg">
        <property name="text">
         <string>Image Thumbnail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPDF">
        <property name="text">
         <string>PDF Thumbnail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNet">
        <property name="text">
         <string>NetworkImage Thumbnail</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cdocumentthumbnaildemo.qrc"/>
 </resources>
 <connections/>
</ui>
